### DISCUSSION TITLE
How to use for metadata only (I want audio transfer completely disabled)

### DISCUSSION THREAD

**Discussion by kylejohnsonkj** [mikebrady/shairport-sync#1304] [Q&A] [answered]
I have a raspberry pi connected to an led matrix and am using this project to airplay album artwork to the display. However, I noticed that when music is playing, I am getting interference caused by cpu usage that causes the matrix to flicker. There is none when the music is paused.

I have no sound devices enabled in my setup (solely grabbing album art and that's it), so I was wondering if there was a way to bypass the cpu intensive task of streaming the audio. Or is that required to keep the airplay session alive? All I really want are updates when the song changes, so I can retrieve the new metadata. 

Is this possible?

**Comment by kylejohnsonkj** [comment, marked as answer]
Don't know how to delete this, so I'll just mention that switching the output backend to pipe stopped the noticeable flickering. 👍

**Reply by mikebrady** [reply]
It's always interesting what people are using Shairport Sync for!
